Nightwatch.Resources/Shell.fs

```diff
@@ -10,10 +10,9 @@ open Nightwatch.Core.Resources
 
 let private create (processController : Process.Controller)
                    (param : IDictionary<string, string>) =
-    let commandParams = param.["cmd"].Split(' ')
-    let len = commandParams.Length
-    let command = commandParams.[0]
-    let args = Array.sub commandParams 1 (len - 1)
+    let command = param.["cmd"]
+    let separator = [|' '|]
+    let args = param.["args"].Split(separator, StringSplitOptions.RemoveEmptyEntries)
```

Comment:
That won't work if `args` isn't defined (= `null`). Please add a `null` check. I want the shell resource to work without args (e.g. just `uptime` check or something like that).